include/beman/inplace_vector/inplace_vector.hpp

```diff
@@ -74,7 +74,7 @@ struct inplace_vector_destruct_base {
   inplace_vector_destruct_base(
       const inplace_vector_destruct_base
           &&other) noexcept(std::is_nothrow_move_constructible_v<T>)
-      : elems(), size_(other.size()) {}
+      : elems(), size_(other.size_) {}
```

Comment:
Could you remove this change just to keep the scope of this PR to CMake/infrastructure?

Reply:
Then it does not compile!